notebooks/250-efficient-object-detection/250-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>information is slightly outdated. tensoflow models is also supported by direct loading now. In order to not overwrite this cool notebook. Can we remove this sentence and just say something like "For TF models in Saved Model format, we can convert it to OpenVINO IR format before"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>